src/ui/public/timepicker/pretty_interval.js

```diff
@@ -17,29 +17,39 @@
  * under the License.
  */
 
+import { i18n } from '@kbn/i18n';
+
 const MS_IN_SECOND = 1000;
 const MS_IN_MINUTE = 60 * MS_IN_SECOND;
 const MS_IN_HOUR = 60 * MS_IN_MINUTE;
 const MS_IN_DAY = 24 * MS_IN_HOUR;
 
 export function prettyInterval(intervalInMs) {
+  let interval;
+  let units;
   if (intervalInMs === 0) {
-    return 'Off';
+    return i18n.translate('common.ui.timepicker.off', { defaultMessage: 'off' });
```

Comment:
This went from `Off` to `off` (lower cased). Was that intentional?

Reply:
Fixed